### ISSUE TITLE
Bad allocation and cyclic interaction on skyrim.esm found in LOOT that only happens using MO

### ISSUE THREAD

**Issue by l1qu1d101** [TanninOne/modorganizer#437]
Hi there hopefully you're not too busy with MO2 to help fix an annoying and tricky bug that shows itself when I try to sort with LOOT. 

First of all i'll point you to my topic on github as we've covered some ground -> https://github.com/loot/loot/issues/514. The only issue I have noticed with MO is that with most applcations I run from MO (like LOOT or wrye or GenFNIS etc) after I've finished and have closed them I come back to MO and my Skyrim.esm is unticked. But when I run LOOT I am not able to sort and get told I have cyclic interaction w

**Comment by GSDFan** [comment]
Make sure that "Force-enable game files" is ticked under settings, workarounds tab. That should keep the Skyrim.esm enabled.

**Comment by l1qu1d101** [comment]
Yep already done, is that a bug with my mod organizer or does that happen by programming?

**Comment by l1qu1d101** [comment]
Its as if MO is giving my skyrim.esm a different crc as when I run loot without MO and base esm,esp and bsa's  in data folder I get no issue but as soon as I make a folder in MO's mods folder and copy the esms into there then run loot from mo this error appears. I've also made sure(just in case) there is no other mod overwriting skyrim.esm in MO's data view in the right pane. There is no reason for loot to be giving this error only when I use MO. I have verified my fies using steam a few times already to make sure I have the original esm as well, but I don't always run steam while i'm playing as it takes up memory but I can execute skyrim at any time with the original files if that would help?

**Comment by TanninOne** [comment]
that's probably because when you run with MO you use C:\Program Files\Mod Organizer\mods\Skyrim ALL DLCs + Update + SKSE\Skyrim.esm whereas without MO you get C:\Program Files\Steam\steamapps\common\Skyrim\DATA\Skyrim.esm. Those files are probably different.
If those files were the same they would have the same crc, no reason why MO would affect that. Actually, MO can't affect file content it only influences file lookup (that is: which files get discovered and opened).

Sorry I can't be of more help, but MO really doesn't do anything else, once it handed a file handle to the host application (loot in this case) it's done.

**Comment by l1qu1d101** [comment]
> that's probably because when you run with MO you use C:\Program Files\Mod Organizer\mods\Skyrim ALL DLCs + Update + SKSE\Skyrim.esm whereas without MO you get C:\Program Files\Steam\steamapps\common\Skyrim\DATA\Skyrim.esm. Those files are probably different.

but honestly, all I am doing is copying the esm from one directory to another. I have skyrim.esm in my C:\Program Files\Steam\steamapps\common\Skyrim\DATA\ then run loot i get no errors but when I copy to C:\Program Files\Mod Organizer\mods\Skyrim ALL DLCs + Update + SKSE\ and run loot then i get this error, i've even tried this multiple times in case I made some kind of mistake myself but it does the same every time. 

If I were to keep the Mod Organizer/profiles dir would I have any problems uninstalling then reinstalling MO akin to mods needing reinstalled?
